saleor/webhook/payloads.py

```diff
@@ -1154,6 +1160,187 @@ def generate_excluded_shipping_methods_for_checkout_payload(
     return json.dumps(payload, cls=CustomJsonEncoder)
 
 
+@traced_payload_generator
+def generate_checkout_payload_for_tax_calculation(
+    checkout_info: "CheckoutInfo",
+    lines: Iterable["CheckoutLineInfo"],
+):
+    checkout = checkout_info.checkout
+    included_taxes_in_prices = include_taxes_in_prices()
+    discount_infos = fetch_active_discounts()
+
+    serializer = PayloadSerializer()
+
+    checkout_fields = (
+        "currency",
+        "metadata",
+    )
+
+    # Prepare checkout data
+    address = checkout_info.shipping_address or checkout_info.billing_address
+    # Function `serializer.serialize` assume that each object have `id`.
+    # We could provide our own name as ID field but it changed also the name of field
+    # on the output from this function. We want to have the `id` field in the output
+    # with the global ID of this object.
+    checkout.id = checkout.token  # type:ignore
```

Comment:
Why we don't extend `serializer.serialize` to allow provide a field name for PK? It looks like a workaround currently but we move a lot of objects to the new approach, where `token` is PK. In that case, it means that we will have to do similar thing for the rest of the payloads.